index.html

```diff
@@ -29,5 +31,113 @@ <h1>I'm Leonardo Glad to see you!</h1>
+                <li>html</li>
+                <li>css</li>
+                <li>javaScript</li>
+            </ul>
+            <button class="work-btn" type="button">See Project</button>
+        </div>
+        <div class="container">
+            <img class="work-img" src="images/SnapshootPortfolio(1).png" alt="a">
+            <h2>Tonic</h2>
+            <ul class="work-information">
+                <li class="canopy">CANOPY</li>
+                <li><span class="dot"></span></li>
+                <li>Back End Dev</li>
+                <li><span class="dot"></span></li>
+                <li>2015</li>
+            </ul>
+            <p class="work-description">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            <ul class="work-tags">
+                <li>html</li>
+                <li>css</li>
+                <li>javaScript</li>
+            </ul>
+            <button class="work-btn" type="button">See Project</button>
+        </div>
+        <div class="container">
+            <img class="work-img" src="images/SnapshootPortfolio(2).png" alt="a">
+            <h2>Tonic</h2>
+            <ul class="work-information">
+                <li class="canopy">CANOPY</li>
+                <li><span class="dot"></span></li>
+                <li>Back End Dev</li>
+                <li><span class="dot"></span></li>
+                <li>2015</li>
+            </ul>
+            <p class="work-description">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            <ul class="work-tags">
+                <li>html</li>
+                <li>css</li>
+                <li>javaScript</li>
+            </ul>
+            <button class="work-btn" type="button">See Project</button>
+        </div>
+        <div class="container">
+            <img class="work-img" src="images/SnapshootPortfolio(3).png" alt="a">
+            <h2>Tonic</h2>
+            <ul class="work-information">
+                <li class="canopy">CANOPY</li>
+                <li><span class="dot"></span></li>
+                <li>Back End Dev</li>
+                <li><span class="dot"></span></li>
+                <li>2015</li>
+            </ul>
+            <p class="work-description">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            <ul class="work-tags">
+                <li>html</li>
+                <li>css</li>
+                <li>javaScript</li>
+            </ul>
+            <button class="work-btn" type="button">See Project</button>
+        </div>
+    </section>
+    <section id="about">
+        <h1>About Myself</h1>
+        <p class="subtitle">Hello I'm a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don't hesitate to contact me.</p>
+        <a href="#" class="connect-link">LET'S CONNECT</a>
```

Comment:
- [ ] You should use a header tag (hx) for this element. 
```suggestion
        <a href="#" class="connect-link"><h3>LET'S CONNECT</h3></a>
```

Reply:
✔️ This issue is successfully addressed.